### ISSUE TITLE
Name package with driver version

### ISSUE THREAD

**Issue by staze** [maurice-daly/DriverAutomationTool#274]
Would be nice to name the package with the dell driver version number. So instead of naming the package "Dell-Latitude-7420-Windows10" (for example), it would be "Dell-Latitude-7420-Windows10-A04". Then when we want to pull a new driver set for that model, we don't have to rename the existing Driver package (as it is, we do since the DAT will say "Package already exists").

**Comment by rtruss** [comment]
from what I recall it does check the actual version that is in the console vs what it finds on line.  I know I have had multiple versions show up in the past but now I just keep the most current dell drivers so it does clean references to older drivers now.

**Comment by staze** [comment]
yeah. if you go to update your driver pack, it complains one is there, so you have to delete or rename existing one before you can grab new version. =/
